samples/buildstrategy/buildah/buildstrategy_buildah_cr.yaml

```
@@ -17,9 +17,34 @@ spec:
         - |
           set -euo pipefail
 
+          echo "Creating registries config file..."
+
+          formate(){
+              array=(`echo $1 | tr ',' ' '`)
+              str=""
+              for m in "${array[@]}"; do
+                  str=$str"'${m}', "
+              done
+
+              echo ${str%??}
+          }
+
+          path="/tmp/registries.conf"
+          registry_search=`formate $(params.registry-search)`
+          registry_insecure=`formate $(params.registry-insecure)`
+          registry_block=`formate $(params.registry-block)`
```

I think we need double-quotes on all these places to pass the argument here for the empty strings. This is imo why e2e fails.

```suggestion
          registry_search="$(formate "$(params.registry-search)")"
          registry_insecure="$(formate "$(params.registry-insecure)")"
          registry_block="$(formate "$(params.registry-block)")"
```

---

Done